presto-hive/src/main/java/com/facebook/presto/hive/RetryDriver.java

```diff
@@ -92,6 +92,11 @@ public final RetryDriver exponentialBackoff(Duration minSleepTime, Duration maxS
         return new RetryDriver(maxAttempts, minSleepTime, maxSleepTime, scaleFactor, maxRetryTime, exceptionMapper, exceptionWhiteList, retryRunnable);
     }
 
+    public final RetryDriver exponentialBackoff(int maxAttempts, Duration minSleepTime, Duration maxSleepTime, Duration maxRetryTime, double scaleFactor, Function<Exception, Exception> exceptionMapper, List<Class<? extends Exception>> exceptionWhiteList, Optional<Runnable> retryRunnable)
```

Comment:
This new constructor should not be needed, as the exception mapper can continue to be set with `exceptionMapper()`. Each of these builder methods can be chained to return a new immutable `RetryDriver` with the updated configuration.

Reply:
@electrum  Done